docs/theory/diffusion/diffusion.rst

```diff
@@ -0,0 +1,713 @@
+
+    plt.ylim([-5, 6])
+
+.. image:: figures/short_track_estimates.png
+
+In this case, getting precise per track estimates at low SNR is unrealistic (note the vertical axis range), since all the estimators perform poorly at low SNRs when there are only few points in the track.
+
+However, it might be possible to still get a single good estimate for an ensemble of tracks (analyzing multiple tracks at once).
+With ensemble analysis we assume that the diffusion for the individual tracks is the same.
+How to best aggregate multiple tracks to obtain such an estimate depends depends on the method of choice.
+
+Covariance-Based Estimator
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Aggregating results for the CVE is straightforward, since those can just be obtained by performing a weighted average of the per-track results.
+Here the weights are chosen to be the number of points contributing to each track.
+This way, longer tracks contribute more to the estimate than short tracks which is beneficial for both accuracy and precision.
+The weighted average is computed as :cite:`vestergaard2014optimal`:
+
+.. math::
+
+    \overline{\hat{D}} = \frac{\sum_{m=1}^M N_m \hat{D}_m}{\sum_{m=1}^M N_m}
+
+Here :math:`M` is the number of tracks, :math:`D_m` corresponds to the diffusion constant of track :math:`m` while :math:`N_m` corresponds to the number of data points contributing to its estimate.
+The associated variance of the weighted mean is given by:
+
+.. math::
+
+    \overline{\hat{D}} = \frac{\sum_{m=1}^M N_m \left(\hat{D}_m - \overline{\hat{D}}\right)^2}{\left(M - 1\right) \sum_{m=1}^M N_m}
+
+To check how the performance of the ensemble estimators compares to individual track estimation, the estimation procedure becomes a bit more complex.
+Instead of simulating a single track many times, we simulate a collection of tracks many times and perform estimation on this.
+Unfortunately, this means that these notebook cells also take more time to evaluate::
+
+    def simulate_tracks(snr):
+        """Function used to simulate tracks"""
+        params = snr_to_diffusion_parameters(snr)
+        tracks = [
+            lk.simulation.simulate_diffusive_tracks(**params, steps=8, num_tracks=10)
+            for _ in range(100)
+        ]
+        return tracks, params["diffusion_constant"]
+
+    methods = {
+        "cve single": lambda list_of_tracks: [t.estimate_diffusion("cve") for tracks in list_of_tracks for t in tracks],
+        "cve ensemble": lambda list_of_tracks: [t.ensemble_diffusion("cve") for t in list_of_tracks],
+    }
+
+    snrs = 10**np.arange(-1, 1.1, 0.125)
+    results, variances = test_estimation_methods(simulate_tracks, snrs, methods=methods)
+    for method, estimates in results.items():
+        plot_accuracy(snrs, estimates, variance=variances[method], label=method);
+
+    plt.ylim([-5, 6])
+
+.. image:: figures/cve_ensemble.png
+
+Here we simulated `10` tracks with the same `8` steps as in the case which performed so poorly before.
```

Comment:
Reading down to this point, I already forgot what `8` steps performed poorly before.. Better to just spell out what you are doing here.

Reply:
Fair point! Fixing...